app/code/AKisilenko/ModuleLesson9/view/frontend/templates/widget/homepagebanner.phtml

```diff
@@ -0,0 +1,32 @@
+
+
+<style>
+    .button-homepage {
+        background: <?php echo $block->getData('button_color') ?>;
```

Comment:
- styles need use in css file or use inline style
- use set/get methods like getButtonColor()